test-case/check-fw-echo-reference.sh

```diff
@@ -20,8 +20,7 @@ set -e
 rm -f /tmp/bat.wav.*
 
 # shellcheck source=case-lib/lib.sh
-libdir=$(dirname "${BASH_SOURCE[0]}")
-source "$libdir"/../case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
```

Comment:
I think you could have just moved the shellcheck comment one-line down instead but no big deal.